.env.example

```diff
@@ -8,20 +8,20 @@ PORT=4321
 
 # Run Talawa-api locally in your system, and put its url into the same.
 
-REACT_APP_TALAWA_URL=
+REACT_APP_TALAWA_URL=http://localhost:4000/graphql/
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD .env.example
git commit

Please apply these changes to this file.